planner/core/testdata/integration_serial_suite_out.json

```diff
@@ -153,15 +153,15 @@
         "SQL": "explain format = 'verbose' select count(*) from t1 join t2 on t1.a = t2.a",
         "Plan": [
           "StreamAgg_12 1.00 18.93 root  funcs:count(1)->Column#7",
-          "└─TableReader_44 3.00 9.93 root  data:ExchangeSender_43",
-          "  └─ExchangeSender_43 3.00 235.38 cop[tiflash]  ExchangeType: PassThrough",
-          "    └─HashJoin_40 3.00 235.38 cop[tiflash]  inner join, equal:[eq(test.t1.a, test.t2.a)]",
-          "      ├─ExchangeReceiver_19(Build) 3.00 77.00 cop[tiflash]  ",
-          "      │ └─ExchangeSender_18 3.00 77.00 cop[tiflash]  ExchangeType: Broadcast",
-          "      │   └─Selection_17 3.00 74.00 cop[tiflash]  not(isnull(test.t1.a))",
-          "      │     └─TableFullScan_16 3.00 65.00 cop[tiflash] table:t1 keep order:false",
-          "      └─Selection_21(Probe) 3.00 74.00 cop[tiflash]  not(isnull(test.t2.a))",
-          "        └─TableFullScan_20 3.00 65.00 cop[tiflash] table:t2 keep order:false"
+          "└─TableReader_45 3.00 9.93 root  data:ExchangeSender_44",
```

Comment:
why not brief?

Reply:
This test case is "TestVerboseExplain". I think we should remain `verbose` here since it explicitly says so.